packages/api/src/external/carequality/process-outbound-patient-discovery-resps.ts

```diff
@@ -62,11 +70,57 @@ export async function processOutboundPatientDiscoveryResps({
       },
     });
 
```

Comment:
ditto: with making the below logic reusbale if possible

Reply:
Moved it to a helper function.